src/server/enterprise/server/env.go

```diff
@@ -33,3 +46,36 @@ func EnvFromServiceEnv(senv serviceenv.ServiceEnv, etcdPrefix string) Env {
 		BackgroundContext: senv.Context(),
 	}
 }
+
+func EnterpriseConfigCollection(db *pachsql.DB, listener col.PostgresListener) col.PostgresCollection {
+	return col.NewPostgresCollection(
+		"enterpriseConfig",
+		db,
+		listener,
+		&ec.EnterpriseConfig{},
+		nil,
+	)
+}
+
+func TryEnterpriseConfigPostgresMigration(ctx context.Context, migrationEnv migrations.Env) error {
```

Comment:
Maybe drop the `Try` here.  If it doesn't work then pachd can't come up, so just trying isn't really an option.

Reply:
Also probably worth commenting this function to explain what's going on.  Not everyone has enterprise, some people do, if they do, this migrates it to postgres, etc.